homeassistant/components/sighthound/image_processing.py

```diff
@@ -45,10 +51,14 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         _LOGGER.error("Sighthound error %s setup aborted", exc)
         return
 
+    save_file_folder = config.get(CONF_SAVE_FILE_FOLDER)
+    if save_file_folder:
+        save_file_folder = os.path.join(save_file_folder, "")  # If no trailing / add it
```

Comment:
Use `pathlib.Path` instead. It's a nicer interface than `os.path`.

It seems supported by Pillow:
https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.save

Reply:
Don't add a trailing path separator here. We will use pathlib below too when adding the file to the directory part of the path. That's enough.

Reply:
```suggestion
        save_file_folder = Path(save_file_folder)
```

Reply:
ok